### PR TITLE
Laying Groundwork for Complete Broadening Treatment

### PR DESCRIPTION
**Pull request by smithis7**
**Based on #10 do not merge until #10 is merged**

Creates the necessary functions to eventually have a complete treatment of line broadening. Also puts ``matplotlib notebook`` magic in the notebook (commented out to limit space in the repo).

### CODE REVIEW

File: stardis/opacities/broadening.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+
+def calc_phi(delta_nu, line_id, T, splasma):
```
Comment (jaladh-singhal) [inline stardis/opacities/broadening.py:4]: As per naming conventions in PEP8, we can't start variable names with capital letter. Perhaps, rename it to `temperature`?

File: stardis/opacities/broadening.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    phis : numpy.ndarray
+        Array of shape (no_of_lines_considered, no_of_shells). Line profiles
+        of each line in each shell evaluated at the specified frequency.
+    """
+    lines_nu = splasma.lines.nu.values[::-1]
+    delta_nus = nu - lines_nu[line_id_start:line_id_end]
+    phis = np.zeros((len(delta_nus), len(marcs_model_fv)))
+    
+    for j in range(len(marcs_model_fv)):
+        T = marcs_model_fv.t[j]
```
Comment (jaladh-singhal) [inline stardis/opacities/broadening.py:60]: same here

File: stardis/opacities/broadening.py (hunk 1 above)
Comment (smithis7) [reply stardis/opacities/broadening.py:4]: I will fix this as a part of my fix of issue #13.

File: stardis/opacities/broadening.py (hunk 2 above)
Comment (smithis7) [reply stardis/opacities/broadening.py:60]: See above.

**Pull request merged**